assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -1431,11 +1431,14 @@ export default Component.extend({
     });
   },
 
-  _handle429Errors(error) {
-    if (error?.jqXHR?.status === 429) {
-      popupAjaxError(error);
-    } else {
-      throw error;
+  _handleErrors(error) {
+    switch (error?.jqXHR?.status) {
+      case 429:
+      case 404:
+        popupAjaxError(error);
+        return;
```

Comment:
Not sure it will change any behavior but I think we should use a `break` statement here instead.

Reply:
In this case that won't change anything but I can do this yes :)